diff --git a/contracts/rewards/RewardsDistributor.sol b/gambit_out/mutants/1/contracts/rewards/RewardsDistributor.sol
index 04d057a..9133ab1 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -121,7 +121,8 @@ abstract contract RewardsDistributor is IRewardsDistributor {
   {
     uint256 totalAccrued;
     for (uint256 i = 0; i < _assetsList.length; i++) {
-      totalAccrued += _assets[_assetsList[i]].rewards[reward].usersData[user].accrued;
+      /// AssignmentMutation(`_assets[_assetsList[i]].rewards[reward].usersData[user].accrued` |==> `1`) of: `totalAccrued += _assets[_assetsList[i]].rewards[reward].usersData[user].accrued;`
+      totalAccrued += 1;
     }
 
     return totalAccrued;
