diff --git a/contracts/rewards/RewardsDistributor.sol b/gambit_out/mutants/13/contracts/rewards/RewardsDistributor.sol
index 04d057a..316669c 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -303,7 +303,8 @@ abstract contract RewardsDistributor is IRewardsDistributor {
       indexUpdated = true;
 
       //optimization: storing one after another saves one SSTORE
-      rewardData.index = uint104(newIndex);
+      /// AssignmentMutation(`uint104(newIndex)` |==> `0`) of: `rewardData.index = uint104(newIndex);`
+      rewardData.index = 0;
       rewardData.lastUpdateTimestamp = block.timestamp.toUint32();
     } else {
       rewardData.lastUpdateTimestamp = block.timestamp.toUint32();
