diff --git a/contracts/rewards/RewardsDistributor.sol b/gambit_out/mutants/18/contracts/rewards/RewardsDistributor.sol
index 04d057a..37cf882 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -333,7 +333,8 @@ abstract contract RewardsDistributor is IRewardsDistributor {
     bool dataUpdated;
     if ((dataUpdated = userIndex != newAssetIndex)) {
       // already checked for overflow in _updateRewardData
-      rewardData.usersData[user].index = uint104(newAssetIndex);
+      /// AssignmentMutation(`uint104(newAssetIndex)` |==> `1`) of: `rewardData.usersData[user].index = uint104(newAssetIndex);`
+      rewardData.usersData[user].index = 1;
       if (userBalance != 0) {
         rewardsAccrued = _getRewards(userBalance, newAssetIndex, userIndex, assetUnit);
 
